rclcpp/include/rclcpp/client.hpp

```diff
@@ -67,7 +67,8 @@ class ClientBase
 
   virtual std::shared_ptr<void> create_response() = 0;
   virtual std::shared_ptr<void> create_request_header() = 0;
-  virtual void handle_response(std::shared_ptr<void> &response, std::shared_ptr<void> &req_id) = 0;
+  virtual void handle_response(std::shared_ptr<void> &request_header,
+                               std::shared_ptr<void> &response) = 0;
```

Comment:
We really need to settle on the code style. I dislike this indentation so much...